### ISSUE TITLE
Add tests with Jest

### ISSUE THREAD

**Issue by morajabi** [morajabi/styled-media-query#1]

**Comment by brajabi** [comment]
Maybe I can take this? @morajabi Why did you choose Jest?

**Comment by morajabi** [comment]
Great! I think Jest is simplest method with awesome features out-of-the-box! If you have any questions mention me.

**Comment by timhagn** [comment]
Hope, I haven't stepped on anyone's toes, but I just went ahead and started writing the tests.
@morajabi Maybe you could merge my first PR ahead of them, so you could "test" if your build 
workflow with rollup is really still working with npm.
Had to add react & react-dom (same versions as in styled-components, speaking newest 16.6.0)
to devDependencies for the tests to run.

**Comment by morajabi** [comment]
@timhagn No, you haven't stepped on anyone's toes! 😄🙌

**Comment by timhagn** [comment]
@morajabi Puh, now I'm relieved ^^. Just thought, because @brajabi wanted to do this in July, there might be a conflict of interest ; ).

**Comment by timhagn** [comment]
Is this and my PR #15 stale or still welcome? Also, how shall it interact and be merged with #16 pertaining #3? Questions above questions ^^.

**Comment by morajabi** [comment]
@timhagn Surely welcome, why not? I'm not sure, should we merge #15 first or #16?

**Comment by timhagn** [comment]
Hi @morajabi, 

though I'm quite new to TypeScript, I like the work @franky47 put into #16.
But as was pointed out [in the first comment](https://github.com/morajabi/styled-media-query/pull/16#issue-232640591) of PR #16 maybe #15 should be merged first, methinks.

What do the others think?

(And I just asked if this issue / #15 were now stale, cause of the time lag ; ).

**Comment by timhagn** [comment]
Opened a new PR #19 with updated tests bringing them to 100% coverage : )
